### DISCUSSION TITLE
Dare generator

### DISCUSSION THREAD

**Discussion by Kelztergit** [TehGM/Randominator#9] [Ideas]
I dare you to X, a/an/the/some, Y/Ys

**Comment by TehGM** [comment]
Hey, any special rules? Some examples?
Also, if you got any word list suggestions, I'll take that too.

**Reply by Kelztergit** [reply]
Example idea 1)
Start with a verb : Eat/Steal/Touch/Feel.
Slap in a noun with an indefinite or definite article:  a car/ an apple/the horse
Add an 'adverb of manner' for flavour. : carefully/beautifully/nicely

**Comment by TehGM** [comment]
To keep it easy to find for myself, word lists provided by @Kelztergit on Discord:

https://github.com/datmt/English-Verbs/blob/master/verbsList  
https://github.com/hugsy/stuff/blob/main/random-word/english-nouns.txt

**Comment by TehGM** [comment]
Implemented as of commit 9faaf362886ce6c1339e8128cb58668b9b41ca47, will be released with Randominator v0.3.

**Comment by TehGM** [comment]
Released with Randominator v0.3 (commit 4acfafc45c5d3ac306d522d84b22f1b31d40db3d).
